dask/dask-schema.yaml

```diff
@@ -247,6 +247,11 @@ properties:
   admin:
     type: object
     properties:
+      async-client-fallback:
+        type: [string, 'null']
+        description: |
+          If False, raise an exception if dask.compute is attempting to select
+          an asynchronous Client. If True, fall back to a sync scheduler.
```

Comment:
It looks like this is supposed to be boolean, not a string? Also, what happens on `null`?

Reply:
That description is off